src/scandeval/dataset_configs.py

```diff
@@ -688,6 +688,37 @@ def get_dataset_config(dataset_name: str) -> DatasetConfig:
     unofficial=True,
 )
 
+ICE_EC_CONFIG = DatasetConfig(
+    name="ice-ec",
+    pretty_name="the Icelandic Error Corpus (a subset of the dataset)",
+    huggingface_id="ScandEval/ice-ec",
+    task=LA,
+    languages=[IS],
+    prompt_prefix="Eftirfarandi eru setningar og hvort þær eru málfræðilega réttar.",
+    prompt_template="Setning: {text}\nMálfræðilega rétt: {label}",
+    prompt_label_mapping=dict(correct="já", incorrect="nei"),
+    instruction_prompt="Setning: {text}\n\nGreinið hvort setningin er málfræðilega "
+    "rétt eða ekki. Svarið skal vera 'já' ef setningin er rétt og 'nei' ef hún er ekki.",
+    num_few_shot_examples=12,
+    max_generated_tokens=1,
+    unofficial=True,
+)
+
+ICE_EC_FULL_CONFIG = DatasetConfig(
+    name="ice-ec-full",
+    pretty_name="the full Icelandic Error Corpus",
```

Comment:
Just to make it consistent with the other dataset configs:

```suggestion
    pretty_name="the Icelandic Error Corpus",
```